ivy_tests/test_ivy/test_frontends/test_paddle/test_tensor/test_math.py

```diff
@@ -1498,7 +1498,7 @@ def test_paddle_log2(
 @handle_frontend_test(
     fn_tree="paddle.logit",
     dtype_and_x=helpers.dtype_and_values(
-        available_dtypes=helpers.get_dtypes("float"),
+        available_dtypes=helpers.get_dtypes("float_and_complex"),
```

Comment:
I think the paddle framework itself doesn't support complex so this shouldn't be included (the frontends only need to support the dtypes that the corresponding framework supports)

Reply:
yeah yeah I forgot also about that